roles/openshift_node/test/test_oc_csr_approve.py

```diff
@@ -51,7 +51,7 @@ def test_get_nodes():
         oc_get_nodes_stdout = stdoutfile.read()
 
     module = DummyModule({})
-    approver = CSRapprove(module, 'oc', '/dev/null', [])
+    approver = CSRapprove(module, 'oc', '/dev/null', [], 1)
```

Comment:
No need to change this, the arguments has a default value and that value is `1`.